### ISSUE TITLE
Cannot install dotnet SDK on Ubuntu 21.04

### ISSUE THREAD

**Issue by stepanprokipchyn** [dotnet/sdk#18304]
So, I've been using .net core previously on Ubuntu. It was 3.1. I don't remember how exactly it was installed but it worked perfectly.

Now when I'd reinstalled my system to Ubuntu 21.04 I was going to give .net 5.0 a try. Unfortunately all installation options mentioned on official Microsoft website (https://docs.microsoft.com/en-us/dotnet/core/install/linux) failed for me.
* Snap installation itself worked fine. I was able to install .net 5.0. But almost any dotnet CLI call fails with Segmentation Fault. This seams to be a known issue, some people recommend to not use Snap for dotnet. It worths mentioning that I've also tried 3.1 channel and it gives the same result - segmentation fault.  
* My second option was to use install.sh, but this is not recommended for development environment installation, so I didn't even try.
* Some people on the internet recommend to use Microsoft PPA and to install deb packages. While this option is still present on official documentation (https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#2104-), it seems like Microsoft team abandoned it's PPA. 
https://packages.microsoft.com/config/ubuntu/21.04/packages-microsoft-prod.deb - gives 404. The same happens if I add PPA manually and try to install dotnet
```bash
$ sudo apt install  dotnet-sdk-5.0 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  aspnetcore-runtime-5.0 aspnetcore-targeting-pack-5.0 dotnet-apphost-pack-5.0 dotnet-host dotnet-hostfxr-5.0 dotnet-runtime-5.0 dotnet-runtime-deps-5.0 dotnet-targeting-pack-5.0
  netstandard-targeting-pack-2.1
The following NEW packages will be installed:
  aspnetcore-runtime-5.0 aspnetcore-targeting-pack-5.0 dotnet-apphost-pack-5.0 dotnet-host dotnet-hostfxr-5.0 dotnet-runtime-5.0 dotnet-runtime-deps-5.0 dotnet-sdk-5.0 dotnet-targeting-pack-5.0
  netstandard-targeting-pack-2.1
0 upgraded, 10 newly installed, 0 to remove and 24 not upgraded.
Need to get 95,1 MB of archives.
After this operation, 396 MB of additional disk space will be used.
Do you want to continue? [Y/n] 
Err:1 https://packages.microsoft.com/ubuntu/21.04/prod hirsute/main amd64 dotnet-runtime-deps-5.0 amd64 5.0.7-1
  404  Not Found [IP: 104.214.230.139 443]
Err:2 https://packages.microsoft.com/ubuntu/21.04/prod hirsute/main amd64 dotnet-host amd64 5.0.7-1
  404  Not Found [IP: 104.214.230.139 443]
Err:3 https://packages.microsoft.com/ubuntu/21.04/prod hirsute/main amd64 dotnet-hostfxr-5.0 amd64 5.0.7-1
  404  Not Found [IP: 104.214.230.139 443]
Err:4 https://packages.microsoft.com/ubuntu/21.04/prod hirsute/main amd64 dotnet-runtime-5.0 amd64 5.0.7-1
  404  Not Found [IP: 104.214.230.139 443]
Err:5 https://packages.microsoft.com/ubuntu/21.04/prod hirsute/main amd64 aspnetcore-runtime-5.0 amd64 5.0.7-1
  404  Not Found [IP: 104.214.230.139 443]
Err:6 https://packages.microsoft.com/ubuntu/21.04/prod hirsute/main amd64 dotnet-targeting-pack-5.0 amd64 5.0.0-1
  404  Not Found [IP: 104.214.230.139 443]
Err:7 https://packages.microsoft.com/ubuntu/21.04/prod hirsute/main amd64 aspnetcore-targeting-pack-5.0 amd64 5.0.0-1
  404  Not Found [IP: 104.214.230.139 443]
Err:8 https://packages.microsoft.com/ubuntu/21.04/prod hirsute/main amd64 dotnet-apphost-pack-5.0 amd64 5.0.7-1
  404  Not Found [IP: 104.214.230.139 443]
Err:9 https://packages.microsoft.com/ubuntu/21.04/prod hirsute/main amd64 netstandard-targeting-pack-2.1 amd64 2.1.0-1
  404  Not Found [IP: 104.214.230.139 443]
Err:10 https://packages.microsoft.com/ubuntu/21.04/prod hirsute/main amd64 dotnet-sdk-5.0 amd64 5.0.301-1
  404  Not Found [IP: 104.214.230.139 443]
E: Failed to fetch https://packages.microsoft.com/ubuntu/21.04/prod/pool/main/d/dotnet-runtime-deps-5.0/dotnet-runtime-deps-5.0.7-x64.deb  404  Not Found [IP: 104.214.230.139 443]
E: Failed to fetch https://packages.microsoft.com/ubuntu/21.04/prod/pool/main/d/dotnet-host/dotnet-host-5.0.7-x64.deb  404  Not Found [IP: 104.214.230.139 443]
E: Failed to fetch https://packages.microsoft.com/ubuntu/21.04/prod/pool/main/d/dotnet-hostfxr-5.0/dotnet-hostfxr-5.0.7-x64.deb  404  Not Found [IP: 104.214.230.139 443]
E: Failed to fetch https://packages.microsoft.com/ubuntu/21.04/prod/pool/main/d/dotnet-runtime-5.0/dotnet-runtime-5.0.7-x64.deb  404  Not Found [IP: 104.214.230.139 443]
E: Failed to fetch https://packages.microsoft.com/ubuntu/21.04/prod/pool/main/a/aspnetcore-runtime-5.0/aspnetcore-runtime-5.0.7-x64.deb  404  Not Found [IP: 104.214.230.139 443]
E: Failed to fetch https://packages.microsoft.com/ubuntu/21.04/prod/pool/main/d/dotnet-targeting-pack-5.0/dotnet-targeting-pack-5.0.0-x64.deb  404  Not Found [IP: 104.214.230.139 443]
E: Failed to fetch https://packages.microsoft.com/ubuntu/21.04/prod/pool/main/a/aspnetcore-targeting-pack-5.0/aspnetcore-targeting-pack-5.0.0.deb  404  Not Found [IP: 104.214.230.139 443]
E: Failed to fetch https://packages.microsoft.com/ubuntu/21.04/prod/pool/main/d/dotnet-apphost-pack-5.0/dotnet-apphost-pack-5.0.7-x64.deb  404  Not Found [IP: 104.214.230.139 443]
E: Failed to fetch https://packages.microsoft.com/ubuntu/21.04/prod/pool/main/n/netstandard-targeting-pack-2.1/netstandard-targeting-pack-2.1_2.1.0-1_amd64.deb  404  Not Found [IP: 104.214.230.139 443]
E: Failed to fetch https://packages.microsoft.com/ubuntu/21.04/prod/pool/main/d/dotnet-sdk-5.0/dotnet-sdk-5.0.301-x64.deb  404  Not Found [IP: 104.214.230.139 443]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```

To summarize, I was not able to install ANY version of .net core using any of suggested approaches. So it there any way to install dotnet on Ubuntu?

**Comment by zmj** [comment]
I had the same issue earlier today, but the .deb file just started resolving again (within the last few minutes).

**Comment by adegeo** [comment]
@zmj first off, descent rules.. 

I too just tried and it wasn't working. This was also reported in the docs. dotnet/docs#24706

**Comment by zmj** [comment]
@adegeo I'm getting 404s again myself. Definitely not fixed yet.

Did you ever try Overload? From some of the old Parallax Software folks that made Descent.

**Comment by adegeo** [comment]
@zmj Totally. I was all in that kickstarter. Also, when they come to seattle for PAX to promote the game, I showed up at their booth with my copies of Descent 1-3 and had them sign the boxes 😁 😁

**Comment by adegeo** [comment]
@rbhanda FYI

**Comment by rbhanda** [comment]
There is an outage going with the infrastructure team hence an issue with packages.microsoft.com. This is expected to be resolved in next hour or so.

**Comment by PeterHagen** [comment]
I still get a 404 error on attempting to download the ubuntu or Debian packages-microsoft-prod.deb file. It does show up in de directory index.

**Comment by ozonni** [comment]
same here for Ubuntu 18.04: https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb

![image](https://user-images.githubusercontent.com/25206121/122368164-62b6fe00-cf5d-11eb-88f1-0169d5666b91.png)

**Comment by stepanprokipchyn** [comment]
> same here for Ubuntu 18.04: https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb
> 
> ![image](https://user-images.githubusercontent.com/25206121/122368164-62b6fe00-cf5d-11eb-88f1-0169d5666b91.png)

It if for all of the version. It is like the deb packages were eradicate from the Internet. Even if you google the filename - you won't find any mirror to download them from

**Comment by stepanprokipchyn** [comment]
FYI, here is the snap problem: https://github.com/dotnet/sdk/issues/11639
As you see people suggest to use apt-get (which is obviously not working now) or pacman. Has anyone tried pacman installation? I don't even know how to install the pacman itself on Ubuntu

**Comment by ozonni** [comment]
Seems like deb file is back but still install fails

```
Err:6 https://packages.microsoft.com/ubuntu/16.04/prod xenial InRelease
  Connection timed out after 120000 milliseconds
Reading package lists...
W: Failed to fetch https://packages.microsoft.com/ubuntu/16.04/prod/dists/xenial/InRelease  Connection timed out after 120000 milliseconds
W: Some index files failed to download. They have been ignored, or old ones used instead.
```

![image](https://user-images.githubusercontent.com/25206121/122437672-37063900-cf9a-11eb-8751-8019cf508e8c.png)

**Comment by stepanprokipchyn** [comment]
On my side downloading is veeeeery slow and fails due to timeout at some point

**Comment by stepanprokipchyn** [comment]
I was finally able to install on my side. It was very slow and it interrupted a few times. Seems like the installed sdk is working, at least I don't get Segfault
